src/main/java/org/opensearch/performanceanalyzer/rca/framework/metrics/RcaVerticesMetrics.java

```diff
@@ -36,6 +36,15 @@
  * metrics added by each RCA vertex in RCA graph. All metrics under this category are RCA specific
  */
 public enum RcaVerticesMetrics implements MeasurementSet {
+    INVALID_OLD_GEN_SIZE("InvalidOldGenSize", "count", Collections.singletonList(Statistics.COUNT)),
+    OLD_GEN_RECLAMATION_INEFFECTIVE(
+            "OldGenReclamationIneffective", "count", Collections.singletonList(Statistics.COUNT)),
+    OLD_GEN_CONTENDED("OldGenContended", "count", Collections.singletonList(Statistics.COUNT)),
+    OLD_GEN_OVER_OCCUPIED(
+            "OldGenOverOccupied", "count", Collections.singletonList(Statistics.COUNT)),
```

Comment:
nit: Can we have a space between each one (just to be uniform)?

Reply:
Ack, Done.